virttest/virsh.py

```diff
@@ -1577,8 +1577,13 @@ def undefine(name, options=None, **dargs):
     :return: CmdResult object
     """
     cmd = "undefine %s" % name
-    if options is not None:
-        cmd += " %s" % options
+
+    vmxml = dumpxml(name).stdout_text
```

Comment:
I assume we don't need this change due to --nvram, there is some example how to handle --nvram under current implemenation in file libvirt/tests/src/virsh_cmd/domain/virsh_undefine.py,

Reply:
The implementation in virsh_undefine.py is so complicated and it will affect all other scripts which calls this function. So I directly update to x86_64 to cover seabios which will not have side effect.